LICENSE.md

```diff
@@ -1,81 +0,0 @@
-                           CISCO SAMPLE CODE LICENSE
```

Comment:
We should find what the template's license is and add it instead, I believe.

Reply:
@adamweeks - Looking to know your thoughts on this one

Reply:
@mkesavan13 I believe @ashjorda has updated this repo with the correct license file needed.

Reply:
@Shreyas281299 - In which case, we shouldn't remove this file. Please add it back

Reply:
@ashjorda  has already removed this duplicate lisence.md.